vm/src/vm/runners/builtin_runner/signature.rs

```diff
@@ -225,8 +219,9 @@ impl SignatureBuiltinRunner {
                 (
                     *k,
                     (
-                        Felt252::from_bytes_be(&v.r.to_bytes_be()),
-                        Felt252::from_bytes_be(&v.s.to_bytes_be()),
+                        // TODO: REMOVE UNWRAP
+                        Felt252::from_bytes_be(&v.r.to_bytes_be()).unwrap(),
```

Comment:
I think we can remove this unwrap()

Reply:
I replaced it with an `unwrap_or_default`.